src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
```

Comment:
Components in this repo are basic building blocks for UI and should avoid reliance on translation utils.
I would argue it's not even necessary to use translation utils in the story. All the component and story need to know about `placeholder` is that it is a string.